src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackagesErrorBar.xaml

```diff
@@ -1,24 +0,0 @@
-<UserControl x:Class="NuGet.PackageManagement.UI.PackagesErrorBar"
```

Comment:
Removing this file has caused a build error [here](https://github.com/NuGet/NuGet.Client/blob/6fcbd086faa7013b9b09e5b0f36e78059d1198bc/src/NuGet.Clients/NuGet.PackageManagement.UI/NuGet.PackageManagement.UI.csproj#L114). Please fix that.

Reply:
yeah saw that after the push, need to remove this file from csproj